### PR TITLE
Removed GenerationConfig settings that are unrelated to safety

### PR DESCRIPTION
**Pull request by Thoughtseize1**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: generative_ai/gemini_safety_config_example.py [hunk 1]
```diff
@@ -40,18 +37,6 @@ def generate_text() -> str:
         max_output_tokens=2048, temperature=0.4, top_p=1, top_k=32
     )
 
-    # Safety config
```
Comment (BigBlackWolf) [inline generative_ai/gemini_safety_config_example.py:43]: Safety config stands on the idea to show how safety settings works. If you remove them, the sample won't make sense. 
The request was to remove unrelated stuff few lines above
```
generation_config = GenerationConfig(
        max_output_tokens=2048, temperature=0.4, top_p=1, top_k=32
)
```
Which has nothing to do with safety, but just adds additional unnecessary complexity.
Please roll back these changes and remove **GenerationConfig**

Comment (Thoughtseize1) [reply generative_ai/gemini_safety_config_example.py:43]: Thanks! Got it

File: generative_ai/gemini_safety_config_example.py [hunk 2]
```diff
@@ -23,23 +23,17 @@ def generate_text() -> str:
...
-        SafetySetting
+        SafetySetting,
     )
 
     # TODO(developer): Update project
     vertexai.init(project=PROJECT_ID, location="us-central1")
 
     model = GenerativeModel("gemini-1.5-flash-001")
 
-    # Generation config
```
Comment (msampathkumar) [inline generative_ai/gemini_safety_config_example.py:38]: (request) Let us drop this! Using Safety config with temperate, top_p and top_k make a lot of sense me during content generation. 

Also, other code samples are are also having same format.

**Review by holtskinner — APPROVED** [review]
